.github/workflows/sanity.yml

```diff
@@ -90,11 +91,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        KUBERNETES_VERSIONS: ["1.20.2"]
+        kubernetes_version: ["1.20.7"]
+        include:
+          - kubernetes_version: "1.20.7"
+            kind_image: "1.20.7@sha256:cbeaf907fc78ac97ce7b625e4bf0de16e3ea725daf6b04f930bd14c67c671ff9"
```

Comment:
Set to a specific hash as per kind documentation.

Ref.: https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1

```
New Node images have been built for kind v0.11.1, please use these exact images (IE like
kindest/node:v1.21.1@sha256:fae9a58f17f18f06aeac9772ca8b5ac680ebbed985e266f711d936e91113bad
including the digest) or build your own as we may need to change the image format again in the
future sweat_smile
```